### PR TITLE
Add MenuItemAttach.

### PR DESCRIPTION
**Pull request by GF-Huang**
#### Features:
- Make menu items of the same level and the same group name work like radio buttons. 
- Disable the checkable menu item to uncheck itself if it is already checked.

#### Example:

image

JVag7sfPIr

### CODE REVIEW

File: src/Shared/HandyControl_Shared/Controls/Attach/MenuItemAttach.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                }
+            }
+        }
+
+        private static void MenuItem_Click(object sender, RoutedEventArgs e)
+        {
+            // prevent uncheck when click the checked menu item
+            if (e.OriginalSource is MenuItem menuItem && !menuItem.IsChecked)
+            {
+                menuItem.IsChecked = true;
```
Comment (ghost1372) [inline src/Shared/HandyControl_Shared/Controls/Attach/MenuItemAttach.cs]: Why shouldn't we be able to uncheck the item?

**Review by ghost1372 — CHANGES_REQUESTED** [review]
Why shouldn't we be able to uncheck the item?

File: src/Shared/HandyControl_Shared/Controls/Attach/MenuItemAttach.cs (hunk 1 above)
Comment (GF-Huang) [reply src/Shared/HandyControl_Shared/Controls/Attach/MenuItemAttach.cs]: This make the same group menu items work like a radio button group, you can't uncheck a checked radio button except you check another one in the same group.

Comment (ghost1372) [reply src/Shared/HandyControl_Shared/Controls/Attach/MenuItemAttach.cs]: Oh I got it👍

**Review by ghost1372 — APPROVED** [review]

**Comment by CodingOctocat** [comment]
补充以下说明：必须要同时设置 `IsCheckable="True"` 才能正确启用 MenuItemAttach，如果没有设置或者 `IsCheckable="False"`，那么 MenuItem 仍然可以 check，但是没有 radio 效果。

**Pull request merged**